packages/swing-store/src/swingStore.js

```diff
@@ -224,14 +231,14 @@ function makeSwingStore(dirPath, forceReset, options) {
     assert.typeof(start, 'string');
     assert.typeof(end, 'string');
 
-    ensureTxn();
-    const cursor = new lmdb.Cursor(txn, dbi);
-    let key = start === '' ? cursor.goToFirst() : cursor.goToRange(start);
-    while (key && (end === '' || key < end)) {
-      yield key;
-      key = cursor.goToNext();
-    }
-    cursor.close();
+    /** @type {import("lmdb").RangeOptions} */
+    const rangeOptions = {};
+    if (start) rangeOptions.start = start;
```

Comment:
I'm not sure why our linter didn't pick this up, but please write this as
```
if (start) {
  rangeOptions.start = start;
}
```
-- always use braces
-- no putting the if consequence on same line as the conditional